client/components/layout/Container/index.js

```diff
@@ -0,0 +1,14 @@
+import React, { PropTypes } from 'react'
+
+import Base from '../internal/Base'
+
+const propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string
+}
```

Comment:
The intent of prop types here is just for documentation of which props are intended to be used on each component, which is why they're different from one to another.